### PR TITLE
[build] Build fix for libstdc++.

### PR DESCRIPTION
**Pull request by ygorshenin**
@rokuz @mgsergio PTAL

### CODE REVIEW

**Comment by rokuz** [comment]
LGTM

File: base/string_utils.hpp [hunk 1]
```diff
@@ -315,6 +316,8 @@ Collection<string> Tokenize(string const & str, char const * delims)
   return c;
 }
 
+static_assert(is_same<vector<string>, decltype(strings::Tokenize("", ""))>::value, "");
```
Comment (mgsergio) [inline base/string_utils.hpp]: "Tokenize should return vector<string> by default."

Comment (ygorshenin) [inline base/string_utils.hpp]: Done

**Comment by ygorshenin** [comment]
PTAL

**Comment by mgsergio** [comment]
LGTM

**Pull request merged**